src/queryBuilder/Rule.js

```diff
@@ -17,6 +17,7 @@ let Rule = ({
   Picker,
   connectDragSource,
   isDragging,
+  ...args
```

Comment:
by convention we usually call this `props` instead of `args` when its a react component